app/assets/stylesheets/application.scss

```diff
@@ -85,7 +86,10 @@ $govuk-use-legacy-palette: false;
 @import 'views/specialist-document';
 @import 'views/answer';
 @import 'views/help-page';
-@import "views/guide";
+@import 'views/guide';
+@import 'views/get-involved';
+@import 'views/headings';
+@import 'helpers/featured_news';
```

Comment:
This seems quite a strange change to include with the route addition. I guess adding the route caused the asset compilation to fail on these imports, hence why they were added. But I think it would be better to add these in earlier on, when the associated CSS was added.